indigo_app/views/tasks.py

```diff
@@ -202,3 +202,68 @@ def post(self, request, *args, **kwargs):
         task.save()
 
         return redirect('task_detail', place=self.kwargs['place'], pk=self.kwargs['pk'])
+
+
+class WorkflowCreateView(TaskViewBase, CreateView):
+    # permissions
+    permission_required = ('indigo_api.add_workflow',)
+
+    js_view = ''
+
+    context_object_name = 'workflow'
+    form_class = WorkflowForm
+    model = Workflow
+
+    def get_context_data(self, *args, **kwargs):
+        context = super(WorkflowCreateView, self).get_context_data(**kwargs)
+        if self.locality:
```

Comment:
Is it possible to combine into `tasks = self.place.tasks`?

Also, do you want to restrict to non-closed tasks? I think I added a method for that on `Task.objects`